website/content/commands/peering/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+layout: commands
+page_title: 'Commands: Peering'
+---
+
+# Consul Peering
+
+Command: `consul peering`
+
+The `peering` command provides management of cluster peering connections, including generating and
+consuming peering tokens.
+[Create and Manage Peerings Connections](/docs/connect/cluster-peering/create-manage-peering) provides an
+overview of the CLI workflow for cluster peering.
+This was added in Consul 1.14.
+
+## Usage
+
+```text
+Usage: consul peering <subcommand> [options]
+
+  # ...
+
+Subcommands:
+
+    delete          Close and delete a peering connection
+    establish       Consume a peering token and establish a connection with the accepting cluster
+    generate-token  Generate a peering token for use by a dialing cluster
+    list            List peering connections of this cluster
```

Comment:
```suggestion
    list            List the local cluster's peering connections
```
Not sure if the active voice is worth it in this spot -- I could go either way.

Reply:
Doesn't hurt!